src/outbackcdx/Webapp.java

```diff
@@ -246,7 +248,14 @@ Response post(Web.Request request) throws IOException {
                         added++;
                     } else {
                         try  {
-                            batch.putCapture(Capture.fromCdxLine(line, canonicalizer));
+                            Capture capture = Capture.fromCdxLine(line, canonicalizer);
+
+                            // remove the protocol, if applicable
+                            if(this.stripDigestScheme && capture.digest.contains(":")) {
+                                capture.digest = capture.digest.split(":")[1];
+                            }
```

Comment:
As mentioned in previous comment I'd prefer if we just enable this for everyone rather than adding an option for it. Then this logic can go in `Capture.fromCdxLine` as that's where all the other field parsing and variant handling logic lives.